commit d06b8d9b156026efebaa9c465f8eb6bea3f2ae1e (refs/changes/49/3849/3)
Author: Kody A Kantor <kody@kkantor.com>
Date:   2018-04-20T16:28:57+00:00 (1 year, 6 months ago)
    
    MANTA-3624 create pgstatsmon image
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/targets.json.in b/targets.json.in
index accbc08..40eeff2 100644
--- a/targets.json.in
+++ b/targets.json.in
@@ -1438,9 +1438,11 @@ cat <<EOF
     ],
     "public": true,
     "deps": [
+      "amon",
       "config-agent"
     ],
     "tarballs": [
+      {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
       {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
     ]
   },
